### PR TITLE
Adding pad_invoice and pad_activation_mails

### PR DESCRIPTION
**Pull request by saravanpa-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/5352
https://github.com/bcgov/entity/issues/5564
https://github.com/bcgov/entity/issues/4855

*Description of changes:*

- Added new email templates and necessary things
- Published a new event from account events to mailer.It needed some code changes in auth as well.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: queue_services/events-listener/requirements.txt [hunk 1]
```diff
@@ -16,12 +16,13 @@ protobuf==3.14.0
...
+sentry-sdk==0.19.5
 six==1.15.0
 threadloop==1.0.2
 thrift==0.13.0
 tornado==6.1
 urllib3==1.26.2
 -e git+https://github.com/bcgov/lear.git#egg=entity_queue_common&subdirectory=queue_services/common
 -e git+https://github.com/bcgov/sbc-common-components.git#egg=sbc-common-components&subdirectory=python
--e git+https://github.com/bcgov/sbc-auth.git@development#egg=auth-api&subdirectory=auth-api
+# -e git+https://github.com/bcgov/sbc-auth.git@development#egg=auth-api&subdirectory=auth-api
```
Comment (mengdong19) [inline queue_services/events-listener/requirements.txt:27]: should it be changed back to bcgov?

File: queue_services/events-listener/requirements/repo-libraries.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
 -e git+https://github.com/bcgov/lear.git#egg=entity_queue_common&subdirectory=queue_services/common
 -e git+https://github.com/bcgov/sbc-common-components.git#egg=sbc-common-components&subdirectory=python
--e git+https://github.com/bcgov/sbc-auth.git@development#egg=auth-api&subdirectory=auth-api
+# -e git+https://github.com/bcgov/sbc-auth.git@development#egg=auth-api&subdirectory=auth-api
```
Comment (mengdong19) [inline queue_services/events-listener/requirements/repo-libraries.txt:3]: same as above. should it be changed back to bcgov?

**Review by mengdong19 — CHANGES_REQUESTED** [review]

File: queue_services/events-listener/requirements.txt (hunk 1 above)
Comment (saravanpa-aot) [reply queue_services/events-listener/requirements.txt:27]: the CI part will break if i change it

File: queue_services/events-listener/requirements/repo-libraries.txt (hunk 2 above)
Comment (saravanpa-aot) [reply queue_services/events-listener/requirements/repo-libraries.txt:3]: same comment as above..I will do another PR right after this to fix this...is that acceptable?

Comment (mengdong19) [reply queue_services/events-listener/requirements/repo-libraries.txt:3]: Sounds good.

**Review by mengdong19 — APPROVED** [review]

File: auth-api/src/auth_api/services/queue_publisher.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+
+# Connection and Queue configuration.
+def nats_connection_options(client_name: str):
+    """Return nats connection options."""
+    return {
+        'servers': current_app.config.get('NATS_SERVERS'),
+        'error_cb': error_cb,
+        'closed_cb': closed_cb,
+        'name': client_name or current_app.config.get('NATS_PAYMENT_CLIENT_NAME'),
```
Comment (mengdong19) [inline auth-api/src/auth_api/services/queue_publisher.py:40]: remove comma.

**Pull request merged**